apps/web/src/helpers/api/errors.ts

```diff
@@ -0,0 +1,19 @@
+import { NextResponse } from 'next/server'
+
+export function apiNotFound(msg?: string) {
+  return NextResponse.json(
+    { error: msg || 'Not Found' },
+    {
+      status: 404,
+    },
+  )
+}
```

Comment:
nextjs has some notFound helper functions but those return ui, not json. Implemented these two for our api. More can be added ofc.